src/language_negotiation_lambda.js

```diff
@@ -23,8 +23,7 @@ exports.handler = (event, context, callback) => {
                 // expected format: q=0.8
                 sanitizedWeight = parseFloat(weight.substring(2)) || 0.0;
             }
-
-            if (supportedLanguages.includes(sanitizedLocale)){
+            if (supportedLanguages.indexOf(sanitizedLocale) >= 0){
```

Comment:
`includes` is not supported by all versions of Node (including the one we use @ edge)